Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM debian:bullseye
+FROM debian:bookworm
 
-ARG FPM_VERSION=1.14.0
+ARG FPM_VERSION=1.15.1
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
       binutils \
       build-essential \
       cpio \
       git \
-      python-setuptools \
+      python3 \
+      python-is-python3 \
```

Comment:
I don't like `python-is-python3`. IMO the proper solution is to fix the code to use `python3`. That's what distributions kind of agree upon. Python2 is dead and with it is `python`. having `python-is-python3` installed just keeps the bad practice alive.